community-concepts/productaanvraag/productaanvraag-denhaag.json

```diff
@@ -0,0 +1,89 @@
+{
+    "title": "Productaanvraag tbv terugbelnotitie Den Haag",
+    "default": {},
+    "examples": [
+        {
+            "data": {
+                "naam": "Jan Jansen",
+                "omschrijving": "Ik heb een vraag over mijn paspoort",
+                "telefoonnummer": "0612345678"
+            },
+            "type": "terugbelnotitie",
+            "bsn": "111222333",
+            "pdf_url": "https://documenten.nl/api/v1/enkelvoudiginformatieobjecten/230bab4a-4b51-40c6-91b2-f2022008a7f8",
```

Comment:
Hmm, this one is ambiguous. Is this a URL to a Documenten API endpoint, or to an actual PDF?

The example at the field shows a different URL then in this example.

Reply:
You're right, it should be a URL to a Documenten endpoint, I changed the examples: https://github.com/open-objecten/objecttypes/pull/3/files#diff-80e76a9d0d1f3cad290e6bd1e5eff75c1e2a003f7ce8be5149100504be6e64a7R60